.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: run tests
+
+on:
+  push:
+    branches: [main, sprint-1, ci-deploy ]
+  pull_request:
+    branches: [ main, sprint-1, ci-deploy ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build --if-present
```

Comment:
Correct me if I'm wrong, but your project doesn't have a `build` script, so I see no reason having it in the CI, even if you're using the `--if-present` flag to handle that it's missing.